### DISCUSSION TITLE
Add a Customizable or Modular Dashboard

### DISCUSSION THREAD

**Discussion by LumosViridi** [twentyhq/twenty#6992] [Ideas]
## Scope & Context

When logged in, create a 'home' page that consists of widgets you can move around and customize. Some data fields that would be nice to have:

- current opportunities or leads,
- expected revenue,
- last months revenue
- tasks or activities that are due today
- anything with a date field within a specified time-frame(we have customer anniversaries stored and it would be nice to know a few days before so I can send an email with a promo).

**Example:** 
**Garmin Dashboard**
![Screenshot 2024-05-05 at 10 05 35](https://github.com/twentyhq/twenty/assets/67052777/006a0d8f-b747-4d37-87b4-a41444174f5a)

**Piperive Insights,** I would love to see more than just financial data though
![Screenshot 2024-05-05 at 10 08 44](https://github.com/twentyhq/twenty/assets/67052777/0b484b1d-47e0-447b-bf82-a4733140dbb5)

**Comment by JuanxCursed** [comment]
+1

**Comment by FelixMalfait** [comment]
Dashboards are now in beta!
